tests/test_slogpy.py

```diff
@@ -1,8 +1,73 @@
 """The test for the slogpy package."""
-
+import pytest
+import re
+from slogpy import slog
 from slogpy import __version__
-
-
+from slogpy.slog import Slog as slog
+from unittest.mock import patch
+import unittest
 def test_version():
     """The most basic of tests."""
     assert __version__ == "0.1.0"
+
+def test_logging_info(capsys):
+
+    slog.info("Log something at the info level")
+    captured = capsys.readouterr()
+     # Remove ANSI escape codes using regex
+    cleaned_output = re.sub(r'\x1b\[[0-9;]*m', '', captured.out)
+    assert cleaned_output == "Log something at the info level\n"
+
+def test_logging_debug(capsys):
```

Comment:
This doesn't really test what is important about debug (that debug output is NOT sent to the console under normal circumstances)